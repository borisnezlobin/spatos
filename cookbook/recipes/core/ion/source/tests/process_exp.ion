echo '# ANCHOR: process_expansion'
mkdir -p _tmp _tmp/t1 _tmp/t2
cd _tmp
let res = $(ls)
let res2 = [ @(ls) ]
echo $res    # output the string
echo @res2     # output the array
cd ..
rm -fr _tmp
echo '# ANCHOR_END: process_expansion'
