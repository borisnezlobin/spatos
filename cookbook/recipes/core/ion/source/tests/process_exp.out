# ANCHOR: process_expansion
t1
t2
t1 t2
# ANCHOR_END: process_expansion
